.github/workflows/run-e2e-tests.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
         go-version: '1.24.5'
 
     - name: Install kubectl, kind and cilium CLI
-      uses: alexellis/arkade-get@master
+      uses: alexellis/arkade-get@67512250416516917cfcda50a3ee1144ce39585a # master
```

Comment:
can you indicate the version in the magic comment nearby with `# vX.Y.Z`, it's for renovate to being able to update it.

Reply:
ah so that's why I didn't use it in the first place... Yeah yeah I see but that would be annoying to have constant renovate updates popping up. Given we no longer use Cilium in the e2e tests after
```
make e2e-test E2E_TESTS=${{matrix.package.f}} E2E_BUILD_IMAGES=0 E2E_AGENT=${{ needs.prepare.outputs.agentImage }} E2E_OPERATOR=${{ needs.prepare.outputs.operatorImage }} EXTRA_TESTFLAGS="-cluster-name=${{ env.clusterName }} -args -v=4 -tetragon.install-cilium=false"
```

maybe we could remove it altogether and install helm/kubectl/kind with our own step or with their respective GitHub action step. It would just need to work both on amd64 and arm64.